.github/workflows/android-build.yml

```
@@ -122,7 +122,43 @@ jobs:
+
+          wget https://android.googlesource.com/platform/sdk/+/98b631c6b3765be96b3707ba7aa54d52b10f4872/files/ant/custom_rules.xml?format=TEXT -O ant/custom_rules.xml.raw \
+            && base64 -d ant/custom_rules.xml.raw > ant/custom_rules.xml
+
+          echo "Installed Ant build tools to $ANDROID_SDK_ROOT/tools/ant/"
+          ls -la $ANDROID_SDK_ROOT/tools/ant/
+
+      - name: Setup Android Ant tasks
+        run: |
+          mkdir -p $ANDROID_SDK_ROOT/tools/ant/etc
+          echo 'sdk.dir=$ANDROID_SDK_ROOT' > proj.android/local.properties
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)